### ISSUE TITLE
Question mark support in English on Windows?

### ISSUE THREAD

**Issue by trypsynth** [RHVoice/RHVoice#969]
On Android using the latest RH Voice app, it sounds like we now have question mark support in English, at least sometimes. However, I can't get it to work in the latest version of the NVDA add-on. Voice in both cases is SLT. Am I missing something, or is this still lacking from the NVDA add-on?

**Comment by rmcpantoja** [comment]
Hi.

We will need to add pitch modification rules in the open-source language pack. The one in the Android app is not open sourced and includes a modifier that partially works, except for some WH questions. Those needs own pitch rules, and it seems that this modifier doesn't have them. Making pitch modificaiton rules for questions involves scheme code, used also in festival, as well as understanding a specific table with pitch values.

I don't know if there are special intonation cases for English; if not, we can rely on pitch rules already made.

Cheers.

**Comment by trypsynth** [comment]
Thanks for your quick reply!
Why is the Android app using a closed-source modifier? Isn't the *entire* point of RH Voice to be a free, open source speech synthesizer?
In either case, I hope to see this soon!

**Comment by zstanecic** [comment]
Hi @rmcpantoja and @trypsynth 
Let me clarify one thing:
Even open-source pack knows how to do proper questions, but only in some sentences. I know it from ukrainian.
The english voices also demonstrate such behavior.

**Comment by trypsynth** [comment]
Hi @zstanecic,
can you give me an example of how to make RH Voice say question marks in English? I can't get it to happen on Windows at all, just on Android.

**Comment by zstanecic** [comment]
@trypsynth I found the cause of the issue. 
Unfortunately, NVDA is nut sending the actual symbol to RHVoice directly. It filters symbols 
You need to go to symbol pronounciation dialog, and change there the rule, so that RHVoice sends the symbol to the tts itself.
Why it is so by default? I have no clue.

**Comment by trypsynth** [comment]
Hi @zstanecic 
My NVDA is set like so for the question mark:
Replacement: question
Level: all
Send actual symbol to synthesizer: always.
I still never hear RH Voice give me question marks on anything. Am I missing something?
